test/common/definitions.spec.js

```diff
@@ -463,6 +463,7 @@ describe('schema definitions', () => {
         city: '    ',
         country: '     ',
         provinceCode: '     ',
+        postalCode: '    ',
```

Comment:
🤔 Does adding this to the test change whether it passes or not? Conversely, would this still fail if we removed the code it should be accounting for? I'm not sure how specific we tend to get with these tests, but from a TDD perspective this feels like it wouldn't add to coverage - the invalid schema is still invalid with the newly invalid field.